repo-scripts/size-analysis/rollup.config.js

```diff
@@ -42,7 +42,7 @@ export default [
         tsconfigOverride: {
           compilerOptions: {
             target: 'es2017',
-            module: 'es2017'
+            module: 'es2015'
```

Comment:
Is this a merge problem?

Reply:
This is a stray change that is unrelated to the TS5 upgrade. 

It turns out that `'es2017'` isn't a valid value for `module`.
```
rpt2: config error TS6046: Argument for '--module' option must be: 'none', 'commonjs', 'amd', 'system', 'umd', 'es6', 'es2015', 'es2020', 'es2022', 'esnext', 'node16', 'nodenext', 'preserve'.
```